e2e/client/client.go

```diff
@@ -78,7 +78,7 @@ type TestClient struct {
 	ks                   *KeyStore
 	logger               zerolog.Logger
 	organizationService  asset.OrganizationServiceClient
-	algoService          asset.AlgoServiceClient
+	functionService          asset.FunctionServiceClient
```

Comment:
Can you please align the right column with the other ones?
```suggestion
	functionService      asset.FunctionServiceClient
```